govcms-ci.Dockerfile

```diff
@@ -20,11 +20,14 @@ RUN apk update \
   && composer clear-cache \
   && rm -rf /app
 
-# Install shellcheck and BATS tools.
-RUN apk update \
-  && apk add --no-cache --repository=http://dl-cdn.alpinelinux.org/alpine/edge/community \
-      bats \
-      shellcheck
+# Install shellcheck.
+RUN curl -L -o "/tmp/shellcheck-v0.7.1.tar.xz" "https://github.com/koalaman/shellcheck/releases/download/v0.7.1/shellcheck-v0.7.1.linux.x86_64.tar.xz" \
```

Comment:
Replaced Shellcheck installation method (no longer can be installed from edge due to conflicts in libs in Alpine 3.10, which is updated in the new base image).